### PR TITLE
Compact dbs manually

### LINKED ISSUE #6575: On Devnet-12 after blobs spamming DB size is significantly bigger than for other ELs
As per EF DevOps:

> Noticed that some of our machines (arms) are running out of disk (devnet 12). Those machines come with very little free disk to begin with, so its very easy to notice. However, looks like there is a pattern about nodes that run nethermind run out of disk a lot quicker than those that don't run nethermind.

> nethermind pairs with free disk space:
> teku: 35GB free
> nimbus: 31GB free
> lodestar: 8GB free
> lighthouse: 4.5GB free

Seems like Geth and Besu are taking only around 20GB of disk space where Nethermind takes from 70-100GB

Looking at detailed db size breakdown:
```
45G blobTransactions
35M blockInfos
34M blockNumbers
3.4G blocks
126M bloom
12M canonicalHashTrie
19M code
1.9M discoveryNodes
124M headers
20M metadata
8.0K peers
1.4G receipts
53G state
12M witness
```

### PR DESCRIPTION
**Pull request by flcl42**
Fixes #6575

No need in resync; very first compaction may take a while.

## Changes

- Compact blobs db manually, as there is no settings found to enable auto-compaction for a column db

#### What types of changes does your code introduce?

- [x] Bugfix (a non-breaking change that fixes an issue)
- [ ] New feature (a non-breaking change that adds functionality)
- [ ] Breaking change (a change that causes existing functionality not to work as expected)
- [ ] Optimization
- [ ] Refactoring
- [ ] Documentation update
- [ ] Build-related changes
- [ ] Other: _Description_

#### Requires testing

- [x] Yes
- [ ] No

#### If yes, did you write tests?

- [ ] Yes
- [x] No

### CODE REVIEW

File: src/Nethermind/Nethermind.Merge.Plugin/BlobTransactionsDbCleaner.cs [hunk 1]
```diff
@@ -55,6 +58,10 @@ private void CleanProcessedTransactionsDb(long newlyFinalizedBlockNumber)
 
             if (_logger.IsDebug) _logger.Debug($"Cleaned processed blob txs from block {_lastFinalizedBlock} to block {newlyFinalizedBlockNumber}");
 
+            _db.Compact();
```
Comment (LukaszRozmej) [inline src/Nethermind/Nethermind.Merge.Plugin/BlobTransactionsDbCleaner.cs:61]: Can we guard the compaction not use to many resources? (Threads? I/O?) So it won't starve anything else in block processing.

Comment (LukaszRozmej) [inline src/Nethermind/Nethermind.Merge.Plugin/BlobTransactionsDbCleaner.cs:61]: Shall we only run compaction only after enough transactions were deleted from the db? Put some threshold?

**Review by LukaszRozmej — APPROVED** [review]
`OnBlocksFinalized` should be between block processing, so should be fine.

**Review by marcindsobczak — APPROVED** [review]

**Comment by flcl42** [comment]
It takes 1-3 seconds to compact every 6 minutes. Seems OK for now.

I'm experimenting with more db options which requires long runs, the only thing that may change is these seconds will be spent by rocksdb in manner opaque for us + maybe less IO spent on compaction.

**Pull request merged**